socs/agents/hwp_pmx/agent.py

```diff
@@ -256,83 +291,87 @@ def acq(self, session, params):
                  'last_updated': 1649085992.719602}
 
         """
+        try:
+            PMX = pmx.PMX(ip=self.ip, port=self.port)
+            self.log.info('Connected to PMX Kikusui')
+        except BrokenPipeError:
+            self.log.error('Could not establish connection to PMX Kikusui')
+            reactor.callFromThread(reactor.stop)
```

Comment:
If we can test the hardware and failure modes, I'd personally opt for something a bit more robust than this. The LS240 restructure branch has an example of how we can structure this such that on connection failures, it will continue to try making the connection instead of just killing the reactor thread: 
https://github.com/simonsobs/socs/blob/adf12ffc29681d51e502fabeb6515d7c3cf64a01/socs/agents/lakeshore240/agent.py#L203-L223